gha.Makefile

```diff
@@ -0,0 +1,127 @@
+ifdef DOCKER_USER
+	DOCKER_IMAGE_NAME ?= ${DOCKER_USER}/actions-runner-controller
+else
+	DOCKER_IMAGE_NAME ?= ghcr.io/actions-runner-controller
+endif
+
+DOCKER_USER ?= $(shell echo ${DOCKER_IMAGE_NAME} | cut -d / -f1)
+
+YQ=$(shell )
+
+IMAGE_VERSION ?= dev
+BINARY_VERSION ?= $(shell cat ./charts/gha-runner-scale-set-controller/Chart.yaml | yq .version)
+
+# default list of platforms for which multiarch image is built
+ifeq (${PLATFORMS}, )
+	export PLATFORMS="linux/amd64,linux/arm64"
+endif
+
+# if IMG_RESULT is unspecified, by default the image will be pushed to registry
+ifeq (${IMG_RESULT}, load)
+	export PUSH_ARG="--load"
+	# if load is specified, image will be built only for the build machine architecture.
+	export PLATFORMS="local"
+else ifeq (${IMG_RESULT}, cache)
+	# if cache is specified, image will only be available in the build cache, it won't be pushed or loaded
+	# therefore no PUSH_ARG will be specified
+else
+	export PUSH_ARG="--push"
+endif
+
+.PHONY: help
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+.PHONY: fmt
+fmt: ## Run go fmt.
```

Comment:
doesn't say anything new that the command doesn't reflect already

Reply:
It serves as a documentation when you just run make
Then it will tell you what does the `make fmt` do (You can possibly have different formatters (gofumpt etc)